### ISSUE TITLE
PyCharmOdooAddonsImportResolver: IndexNotReadyException

### ISSUE THREAD

**Issue by hishamalfoqaha** [wt-io-it/odoo-pycharm-plugin#72]
Plugin Version: 0.6.0
IntelliJ Version: PyCharm 2021.3.3
IntelliJ Build: PY-213.7172.26

<details><summary>com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation
</summary>

```
com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation
	at com.intellij.openapi.project.IndexNotReadyException.create(IndexNotReadyException.java:67)
	...
	at com.intellij.util.indexing.FileBasedIndexEx.getValues(FileBasedIndexEx.java:111)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndex.getValues(OdooIndex.java:49)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModule$1(OdooModuleServiceImpl.java:63)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:852)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModule(OdooModuleServiceImpl.java:61)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:49)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	...
	at com.intellij.refactoring.suggested.SuggestedRefactoringChangeListener$MyDocumentListener.beforeDocumentChange(SuggestedRefactoringChangeListener.kt:210)
	at jdk.internal.reflect.GeneratedMethodAccessor133.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	...
	at com.intellij.util.EventDispatcher.lambda$createMulticaster$1(EventDispatcher.java:85)
	at com.sun.proxy.$Proxy75.beforeDocumentChange(Unknown Source)
	at com.intellij.openapi.editor.impl.DocumentImpl.beforeChangedUpdate(DocumentImpl.java:878)
	...
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
</details>

**Comment by ap-wtioit** [comment]
how to trigger this:
* ~~[CTRL]+[ALT]+[V], convert to variable, is blocked when still indexing ~~
* ~~[SHIFT]+[F6], rename, is blocked when still indexing ~~

**Comment by ap-wtioit** [comment]
@hishamalfoqaha thanks for reporting this, it is fixed in 0.6.7 (pending jetbrains approval) if you are eager to test it you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.7

**Comment by bigbear3001** [comment]
Was fixed (for 2021.3.3) in 0.6.7 available in the marketplace
